bindings/dd/register_dd_package.cpp

```
@@ -14,44 +14,50 @@
 #include "dd/Operations.hpp"
 #include "dd/Package.hpp"
 #include "dd/StateGeneration.hpp"
+#include "ir/Definitions.hpp"
```

Cppcheck (2.18.0)

[information] Include file

(missingInclude)